terraform/backend.local.example

```diff
@@ -0,0 +1,5 @@
+# Copy this file to `backend.local` and update with the S3 bucket name you're using to store
```

Comment:
i think it's better to ask user to provide these variables as part of terraform init process through UI input.

https://learn.hashicorp.com/terraform/getting-started/variables.html

Reply:
I lean towards the config file just because it means I don't have to re-enter the config each time I rerun `terraform init`, which will be required any time we (for example) add a module or change a provider version. It's a slightly increased burden at setup time in exchange for an easier workflow later on.